lib/install/data/data.go

```diff
@@ -89,6 +89,7 @@ func NewData() *Data {
 		MappedNetworksGateways: make(map[string]net.IPNet),
 		MappedNetworksIPRanges: make(map[string][]ip.Range),
 		MappedNetworksDNS:      make(map[string][]net.IP),
+		Timeout:                3 * time.Minute,
```

Comment:
This is a default value. In general, default value can be set in argument default, but if there is no timeout option at all, the default value cannot be set, and then it will be 0 second timeout.
So this is to solve the issue in commands without timeout option.

In addition, the timeout is set for application initialization waiting, not for the whole process, so 3 minutes by default for install is ok as well.